src/main/resources/io/jenkins/plugins/analysis/core/views/SourceDetail/index.jelly

```diff
@@ -2,6 +2,14 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
   <st:header name="Content-Type" value="text/html;charset=UTF-8" />
   <l:layout title="${it.displayName}" norefresh="true">
+    <!-- header start source detail -->
+    <script src="${rootURL}/plugin/warnings/js/libs/prism.js"></script>
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/warnings/css/prism.css"/>
```

Comment:
Is there a way to scroll to the highlighted line?

Reply:
I think I found something, if the primary range is marked with ID focus:
```
    $.fn.scrollView = function () {
        return this.each(function () {
            $('html, body').animate({
                scrollTop: $(this).offset().top - $(window).height()
            }, 1000);
        });
    };
    $('#focus').scrollView();

```

Reply:
Yes, I noticed that as well. I added it now, see https://github.com/jenkinsci/warnings-plugin/commit/f5e1477111b0e703fcd10706f10047e0a285126d.